test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-l10n_eu_product_adr_dangerous_goods @ git+https://github.com/OCA/community-data-files@refs/pull/118/head#subdirectory=setup/l10n_eu_product_adr_dangerous_goods
```

Comment:
To put in a separate TMP commit to warn reviewers that it should be removed before merging

Reply:
And add also `l10n_eu_product_adr` the same way to get tests working.

Reply:
done